File: ffplay/src/main/java/com/mjl/ffplay/NativeLib.java
package com.mjl.ffplay;

public class NativeLib {

    // Used to load the 'ffplay' library on application startup.
    static {
        System.loadLibrary("ffplay");
    }

    /**
     * A native method that is implemented by the 'ffplay' native library,
     * which is packaged with this application.
     */
    public native String stringFromJNI();
}

File: hapirtmp/src/main/java/com/hapi/rtmpliving/NativeLib.kt
package com.hapi.rtmpliving

class NativeLib {

    /**
     * A native method that is implemented by the 'rtmpliving' native library,
     * which is packaged with this application.
     */
    external fun stringFromJNI(): String

    companion object {
        // Used to load the 'rtmpliving' library on application startup.
        init {
            System.loadLibrary("rtmpliving")
        }
    }
}